dissect/target/plugin.py

```diff
@@ -774,7 +795,7 @@ def _modulepath(cls) -> str:
 
 def get_plugin_classes_with_method(method_name: str) -> Iterator[Type[Plugin]]:
```

Comment:
This function is no longer used (only user was `tools/utils.py`) and can be removed. Same for `get_plugin_classes_by_namespace()`